packages/gatsby-transformer-csv/README.md

```diff
@@ -78,6 +78,35 @@ the following three nodes would be created.
 ];
 ```
 
+### Illegal Characters in GraphQL
```

Comment:
I feel like this should be after "How to query" section (instructions before troubleshooting)

Reply:
Agree, this is an end of document thing. Most people want to just get going.